Game/Assets/Shaders/trail.shader

```diff
@@ -22,11 +22,25 @@ in vec2 uv0;
 uniform sampler2D diffuse;
 uniform vec4 inputColor;
 
+uniform float colorFrame;
+uniform vec4 initColor;
+uniform vec4 mediumColor;
+uniform vec4 finalColor;
+
 out vec4 outColor;
 
 void main()
 {	
-	//outColor = inputColor;
-	outColor = texture2D(diffuse, uv0);
+	if(colorFrame<1){
+		float a = colorFrame - int(colorFrame);
+		outColor = mix(initColor, mediumColor, a) * texture2D(diffuse,  uv0 ) ;
+	}
+	else if(colorFrame >= 1  && colorFrame<2 ){
+		float a = colorFrame - int(colorFrame);
+		outColor = mix(mediumColor, finalColor, a) * texture2D(diffuse,  uv0 ) ;
+	}
+	else{
+		outColor = finalColor * texture2D(diffuse,  uv0 ) ;
+	}
```

Comment:
This is pretty good, but we may want to edit the position of the mediumColor. For example:
0-----1-----2
0--0.6------2
0-------1.6-2

Reply:
Agree, we should be able to choose the seconds of duration from each color, also de duration of transition probably. Let's discuss it